### PR TITLE
Routine rubocop fix on /spec

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] Refactor

## Description
Applied `rubocop -A` on the spec folder. All surface level changes.

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: spec/services/articles/feeds/large_forem_experimental_spec.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@
           create(:field_test_membership,
                  experiment: experiment, variant: "base", participant_id: user.id)
           stories = feed.default_home_feed(user_signed_in: true)
-          expect(stories.size).to be > 0
+          expect(stories.size).to be.positive?
```
Comment (jaw6) [inline spec/services/articles/feeds/large_forem_experimental_spec.rb:110]:
```suggestion
          expect(stories.size).to be_positive
```

File: spec/services/articles/feeds/article_score_calculator_for_user_spec.rb [hunk 2]
```diff
@@ -189,7 +189,7 @@
       end
 
       it "returns a non-zero score for commented upon articles" do
-        expect(applicator.score_comments(article_with_one_comment)).to be > 0
+        expect(applicator.score_comments(article_with_one_comment)).to be.positive?
```
Comment (jaw6) [inline spec/services/articles/feeds/article_score_calculator_for_user_spec.rb:192]:
```suggestion
        expect(applicator.score_comments(article_with_one_comment)).to be_positive
```

File: spec/models/async_info_spec.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
     it "has a policies key with an array of policies", :aggregate_failures do
       policies = async_info.fetch(:policies)
-      expect(policies.length).to be > 0
+      expect(policies.length).to be.positive?
```
Comment (jaw6) [inline spec/models/async_info_spec.rb:17]:
```suggestion
      expect(policies.length).to be_positive
```

File: spec/workers/follows/update_points_worker_spec.rb [hunk 4]
```diff
@@ -29,7 +29,7 @@
       worker.perform(reaction.reactable_id, reaction.user_id)
       follow.reload
 
-      expect(follow.implicit_points).to be > 0
+      expect(follow.implicit_points).to be.positive?
```
Comment (jaw6) [inline spec/workers/follows/update_points_worker_spec.rb:32]:
```suggestion
      expect(follow.implicit_points).to be_positive
```

**Review by jaw6 — CHANGES_REQUESTED** [review]
So, I'm not really a fan of the config in rubocop that insists on `positive?`, but the changes I'm suggesting here will get this passing CI. However, if our rubocop is configured to autocorrect this way, it's actively going the wrong direction in `spec` and we should actively exclude that cop in `spec`.

**Comment by maestromac** [comment]
@jaw6 Thanks for that suggested fix. I have no idea what I did prior but after rerunning `rubocop -A` again it did not try to try to `.positive?` on those specs.

**Review by jaw6 — APPROVED** [review]
Somehow, the diff became confusing to read here on GitHub.com, but I pulled the branch locally and it looks fine :+1:

**Pull request merged**